packages/tc-schema-sdk/server.js

```diff
@@ -1,7 +1,7 @@
 const { SDK } = require('./sdk');
 const readYaml = require('./lib/read-yaml');
 
-module.exports = new SDK({ init: false });
+module.exports = new SDK({ init: false, readYaml });
```

Comment:
I just had an idea. You can use `SDK.prototype.readYaml = readYaml;`, then you never have to pass it in explicitly here or in the tests as on any instance of the SDK `this.readYaml` will be inherited from the prototype

Reply:
![giphy](https://user-images.githubusercontent.com/9718606/74817368-ae6b8000-52f4-11ea-91ee-6b48466e46da.gif)